src/effects/effectsmanager.cpp

```diff
@@ -13,6 +13,8 @@
 #include "moc_effectsmanager.cpp"
 #include "util/assert.h"
 
+const QString EffectsManager::kNoEffectString = QStringLiteral(" --- ");
```

Comment:
```suggestion
const QString EffectsManager::kNoEffectString = QStringLiteral("---");
```
?

Reply:
Then it would stick to the left edge (or right edge, depending on how skin desgners align the strings).

Reply:
Shouldn't that be fixed by using padding instead of using spaces for alignment?

Reply:
yes, but the padding would affect all effect names, and I would just like to push the placeholder a bit towards the center, below the other effects.

Reply:
IMHO this looks really weird and might lead to bug reports. I'd prefer just "---" without spaces. Can you elaborate why you want to indent it?

![Screenshot from 2021-01-06 14-58-34](https://user-images.githubusercontent.com/1834516/103776575-eaecbb00-502f-11eb-81b5-c10205aed8ce.png)

Reply:
Same here:
![Screenshot from 2021-01-06 15-02-19](https://user-images.githubusercontent.com/1834516/103776921-723a2e80-5030-11eb-8298-99cc1c18a567.png)

Reply:
I doubt it leads to bug reports, but I don't mind. Initially I was just looking at the empyt effect at the bottom of the list and I didn't want it to look like a bullet point. Interestingly that doesn't look good at the top of the list (sound API & devices).

I'll remove the whitespaces.

Reply:
Done. `---` in effect widgets & prefs